src/client/game.ts

```diff
@@ -129,16 +129,64 @@ export default class ClientGame extends GameLoop {
           player.id,
         );
       } else {
+        this.decide_direction(player.id);
         this.sprite_list[player.id].x = this.states[0].players[
           player.id
         ].position.x;
+
         this.sprite_list[player.id].y = this.states[0].players[
           player.id
         ].position.y;
       }
     }
   }
-
+  decide_direction(player_id: number) {
+    const dx =
+      this.states[0].players[player_id].position.x -
+      this.sprite_list[player_id].x;
+    const dy =
+      this.states[0].players[player_id].position.y -
+      this.sprite_list[player_id].y;
```

Comment:
`this.states[0].players[player_id].position` är en `Vec2` va?
Vi borde göra om this.sprite_list[player_id] så att den också har en position som är av typen `Vec2`, då skulle kunna göra `this.states[0].players[player_id].position.sub(this.sprite_list[player_id].position)` istället

Reply:
tror inte det går va?